File: Dataset/Leetcode/train/136/56.cpp
 class Solution {
public:
    int singleNumber(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int len=nums.size()-1;
        for(int i=0;;i+=2)
            if(i==len||nums[i]!=nums[i+1])
                return nums[i];
        return -1;
    }
};

File: Dataset/Leetcode/test/88/56.py
 class Solution:
    def XXX(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        for i in range(n):
            nums1.remove(0)
        for i in nums2:
            nums1.append(i)
        nums1=nums1.sort()

